libs/langchain/langchain/chat_models/anyscale.py

```diff
@@ -67,12 +74,13 @@ def lc_secrets(self) -> Dict[str, str]:
 
     @staticmethod
     def get_available_models(
-        anyscale_api_key: Optional[str] = None,
+        anyscale_api_key: Optional[SecretStr] = None,
```

Comment:
The static method is public, let's accept `Optional[Union[str, SecretStr]]` and leverage `_to_secret` once again to avoid breaking API for any existing users that rely on the staticmethod directly

Reply:
I've addressed this comment this new commit dcbb17ff89f83fe21394335a8f93c15d903e6fb5